source/documentation/doc-Release_Notes/topics/ref-ovirt-4.5.2.adoc

```diff
@@ -0,0 +1,102 @@
+WARNING: Invalid configuration: 'user_friendly_names' is
+enabled in multipath configuration:
+  section1 {
+    key1 value1
+    user_friendly_names yes
+    key2 value2
+  }
+  section2 {
+    user_friendly_names yes
+  }
+This configuration is not supported and may lead to storage domain corruption.
+
+BZ#link:https://bugzilla.redhat.com/2097536[2097536]::
+In this release, the rhv-log-collector-analyzer now provides a detailed output for each problematic image, including disk names, associated virtual machine, the host running the virtual machine, snapshots, and the current SPM. This makes it easier to identify problematic virtual machines and collect SOS reports for related systems.
+The detailed view is now the default, and the compact option can be set by using the –compact switch in the command line.
+
+BZ#link:https://bugzilla.redhat.com/2097560[2097560]::
+Expiration of ovirt-provider-ovn certificate is now checked regularly along with other RHV certificates (such as engine CA, engine, or hypervisors) and if ovirt-provider-ovn is going to expire or already expired, the warning or alert is raised to the audit log. To renew the ovirt-provider-ovn certificate, administators need to run engine-setup.
+If your ovirt-provider-ovn certificate expires on a previous RHV version, please upgrade to RHV 4.4 SP1 batch 2 or newer, and ovirt-provider-ovn certificate will be renewed automatically as a part of engine-setup.
+
+BZ#link:https://bugzilla.redhat.com/2104939[2104939]::
+With this release, OVA export or import work on hosts with a non-standard SSH port.
+
+BZ#link:https://bugzilla.redhat.com/2107250[2107250]::
+With this release, the process to check certificate validity is now compatible with both RHEL 8 and RHEL 7 based hypervisors.
+
+==== Rebase: Bug Fixes and Enhancements
+
+These items are rebases of bug fixes and enhancements included in this release of Red Hat Virtualization:
+
+BZ#link:https://bugzilla.redhat.com/2092478[2092478]::
+UnboundID LDAP SDK has been rebased on upstream version 6.0.4. Please review https://github.com/pingidentity/ldapsdk/releases for changes since version 4.0.14
+
+==== Rebase: Bug Fixes Only
+
+These items are rebases of bug fixes included in this release of Red Hat Virtualization:
+
+BZ#link:https://bugzilla.redhat.com/2104831[2104831]::
+Rebase package(s) to version: 4.4.7.
+Highlights, important fixes, or notable enhancements: fixed BZ#2081676
+
+==== Release Notes
+
+This section outlines important details about the release, including recommended practices and notable changes to Red Hat Virtualization. You must take this information into account to ensure the best possible outcomes for your deployment.
+
+BZ#link:https://bugzilla.redhat.com/2049286[2049286]::
+With this release, only virtual machines pinned to hosts and selected for upgrade during cluster upgrade are stopped during the cluster upgrade. VMs pinned to other hosts (not selected for upgrade) are left untouched.
```

Comment:
I think you can remove one of the "during cluster upgrade" in the first sentence. The first sentence is ambiguous because it implies that the virtual machines are selected for upgrade. The hosts are selected for upgrade, not the VMs. 

You can fix this by removing "and": "Only virtual machines pinned to hosts selected for upgrade are stopped during the cluster upgrade. VMs pinned to hosts that are not selected for upgrade are not stopped.